pdfarranger/pdfarranger.py

```diff
@@ -267,6 +268,38 @@ def __init__(self, *args, **kwargs):
             # "private" clipboard does not work in Windows so we can't use it here
             self.clipboard_pdfarranger = Gtk.Clipboard.get(Gdk.Atom.intern('_SELECTION_PDFARRANGER',
                                                                            False))
+        if hasattr(self, "add_main_option"):
+            self.add_arguments()
+        else:
+            print(_(
+                "GLib.add_main_option missing, "
+                "was added in GLib 2.40 (Released 2018-09-22)"),
```

Comment:
https://repology.org/project/glib/versions

Reply:
Indeed we don't need that check. I don't think anyone has ever run PDF Arranger with such an old Glib. For Debian it's before 2015 and for Ubuntu the package is not even available anymore.

Reply:
That is correct, moreover we really don't need this check since we `gi.check_version('3.10.2')`. Corrected with commit c27e4332485619aa64c7ce68eed72fffceca67af.